recipe/meta.yaml

```diff
@@ -14,43 +14,60 @@ source:
     - 0001-Rename-setuptools-scm-to-setuptools_scm.patch
 
 build:
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 1
+  # This package isn't included in the SOW Packages
+  skip: True  # [s390x or py>=39]
+  script:
+    - export DISABLE_BCOLZ_AVX2=""  # [unix]
+    - export DISABLE_BCOLZ_SSE2=""  # [ppc64le]
+    - set DISABLE_BCOLZ_AVX2=""  # [win]
+    - python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - patch  # [not win]
+    - m2-patch  # [win]
   host:
     - python
-    - setuptools >18.0
-    - setuptools_scm >1.5.4
     - cython >=0.22
     - numpy
-  build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
+    - pip
+    - setuptools >18.0
+    - setuptools_scm >1.5.4
+    - wheel
   run:
-    - python
+    # It seems like bcolz 1.2.1 isn't compatible with python 3.9+
+    - python <3.9
     - {{ pin_compatible('numpy') }}
-    - numexpr >=2.5.2
-    - dask >=0.9.0
-    - pandas
-    - pytables
+    # Optional dependencies
+    #- numexpr >=2.5.2
+    #- dask >=0.9.0
+    #- pandas
+    #- pytables
```

Comment:
I am open to either approach, however, since we provided this package version in the past with these optional dependencies we should consider keeping them for this build bump.

Reply:
The versions of the pandas are incompatible due to this error:
` AttributeError: type object 'DataFrame' has no attribute 'from_items'`
So tests failed

Reply:
So I enabled only `numexpr` and `pytables` but `pandas` and `dask` (it uses pandas as a dependency) are still commented.

Reply:
I created my patch that fixes pandas Dataframe import from dict. The issue was with this line https://github.com/skupr-anaconda/bcolz/blob/500351db35c7d2a182e1d33e38daccf6a5ac7266/bcolz/tests/test_ctable.py#L2213 and this from tag 1.2.1 https://github.com/skupr-anaconda/bcolz/blob/ce1371774d5ca9445e5331a5838d5cc5bdfd913f/bcolz/ctable.py#L806
But `master` branch already has a fix https://github.com/skupr-anaconda/bcolz/blob/500351db35c7d2a182e1d33e38daccf6a5ac7266/bcolz/ctable.py#L806
So I added a patch and now we use all dependencies, and the build succeeded on all platfroms